src/model/GeneralizedHiddenMarkovModel.cpp

```diff
@@ -30,6 +30,9 @@
 // Interface header
 #include "model/GeneralizedHiddenMarkovModel.hpp"
 
+// Exception header
+#include "exception/NotYetImplemented.hpp"
```

Comment:
Just put it under `// ToPS header`. If there is more than one namespace being included, put the headers from the same namespace first, than the others in alphabetical order, separated by newlines.

E.g.:

```c++
// ToPS headers
#include "model/Generator.hpp"
#include "model/ProbabilisticModel.hpp"

#include "exception/OutOfRange.hpp"
#include "exception/NotYetImplemented.hpp"

#include "helper/DiscreteIIDModel.hpp"

namespace tops {
namespace model {
...
```

In this case, as the implementation is in `namespace tops`, we put all `model` headers first, and then put `exception` and `helper`, with `exception` first because it is the first in alphabetical order.